README.md

```diff
@@ -47,45 +47,24 @@ syntax [here][bazel-cli]. Common [commands][bazel-cli-commands] are
 
 ### Nixpkgs
 
-This rule set supports [Nixpkgs][nixpkgs]. If you are on NixOS, or if
-you are using Nixpkgs on your project, consider passing the following
-argument on the command-line to select a Nixpkgs-based toolchain for
-the build:
+This rule set supports using [Nixpkgs][nixpkgs] to provision your GHC
+toolchain and to fetch hackage packages from there. To create your
+project, instead of passing `--use-bindists`, pass `--use-nix`, like
+so:
 
-```
-$ bazel build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs //...
+```console
+$ sh <(curl https://haskell.build/start) --use-nix
 ```
 
-See [below](#saving-common-command-line-flags-to-a-file) to
-permanently set that flag.
+This generates the same files as above, but uses a `nixpkgs` backend.
```

Comment:
What is a "backend"? It should at the very least be defined, but it's not a term we use anywhere else, so best avoid it.

Reply:
https://github.com/tweag/rules_haskell/pull/1018/commits/3a64a062d2cd87bdf551dbff2c6e5d5864bb8f2c